src/03-dice-roller/DiceRoller/DiceRoller/Dice.cs

```diff
@@ -8,26 +8,22 @@ namespace DiceRoller
 {
     public class Dice
     {
-        public static Dice D6 => new Dice(6, "d6");
-
-        public static Dice D8 => new Dice(6, "d8");
-
+        //class variables
         public int Sides { get; }
+        public string Desc { get; }
 
-        public string Description { get; }
-
-        private static Random Random { get; } = new Random();
-
-        private Dice(int sides, string description)
+        //lets define the dice constructor
```

Comment:
If these comments are useful to yourself (learning purposes) then it's fine, otherwise the comment should say me why, not what.. I can see from the code itself that it is defining a constructor : )